### DISCUSSION TITLE
Dumping NES Games - RAM value?

### DISCUSSION THREAD

**Discussion by muguemjee** [sanni/cartreader#249] [Need help]
When dumping nes games and specifying mapper chr, prg, ram, what is the last value referring to? Is it if the game has battery backup ram? I know its 0 for most games since they dont use a battery but how do i determine what value to put for this for the ones that do?

Or is it referring to another ram value?

**Comment by sanni** [comment]
Yes RAM refers to W-RAM, which is the SRAM chip.

![nesdumpset](https://user-images.githubusercontent.com/19169494/133679871-14f6dfa0-d475-4cb4-832a-d503cdd55ce1.jpg)
